src/vector_windows.rs

```diff
@@ -98,15 +144,15 @@ pub mod service_control {
         Ok(())
     }
 
-    fn stop_service(service_def: &ServiceDefinition) -> Result<()> {
+    fn stop_service(service_def: &ServiceDefinition) -> crate::Result<()> {
         let service_access = ServiceAccess::QUERY_STATUS | ServiceAccess::STOP;
-        let service = open_service(&service_def, service_access)?;
-        let service_status = service.query_status()?;
+        let service = open_service(&service_def, service_access).map_err(Error::from)?;
```

Comment:
Usually it's enough to use `?` here, since that will call `.into()` for you.

Reply:
From what I've understood, in this particular case, `.into()` will actually `box` my error type since `crate::Error` is a `Box`, but won't map the error type from `windows-service` (which is a `windows_service::Error`) to my custom `Error` type, right ?